radio/src/gui/128x64/model_logical_switches.cpp

```diff
@@ -101,7 +101,7 @@ void menuModelLogicalSwitchOne(event_t event)
         lcdDrawTextAlignedLeft(y, STR_V1);
         int v1_min=0, v1_max=MIXSRC_LAST_TELEM;
         if (cstate == LS_FAMILY_BOOL || cstate == LS_FAMILY_STICKY || cstate == LS_FAMILY_EDGE) {
-          drawSwitch(CSWONE_2ND_COLUMN, y, v1_val, attr);
+          drawSwitch(CSWONE_2ND_COLUMN, y, v1_val, attr | ((getSwitch(v1_val) && v1_val) ? BOLD : 0));
```

Comment:
the condition should be reversed: `v1_val && getSwitch(v1_val)`. This avoids evaluating `getSwitch()` for nothing.